homeassistant/components/wevolor/__init__.py

```diff
@@ -0,0 +1,31 @@
+"""The Wevolor Control for Levolor Motorized Blinds integration."""
+from __future__ import annotations
+
+from pywevolor import Wevolor
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS: list[str] = [
+    "cover",
```

Comment:
Use the `Platform` StrEnum from `homeassistant.const` instead of a string.

Reply:
:+1: